econml/inference/_inference.py

```diff
@@ -1309,62 +1299,49 @@ def _print(self, *, alpha=None, value=None, decimals=None, tol=None, output_name
         output_names = self.output_names if output_names is None else output_names
 
         # 1. Uncertainty of Mean Point Estimate
-        res1 = self._res_to_2darray(self._d_t, self.d_y, self.mean_point, decimals)
+        res1 = self._format_res(self.mean_point, decimals)
         if self.pred_stderr is not None:
-            res1 = np.hstack((res1, self._res_to_2darray(self._d_t, self.d_y, self.stderr_mean, decimals)))
-            res1 = np.hstack((res1, self._res_to_2darray(self._d_t, self.d_y, self.zstat(value=value), decimals)))
-            res1 = np.hstack((res1, self._res_to_2darray(self._d_t, self.d_y, self.pvalue(value=value), decimals)))
-            res1 = np.hstack((res1, self._res_to_2darray(self._d_t, self.d_y,
-                                                         self.conf_int_mean(alpha=alpha)[0], decimals)))
-            res1 = np.hstack((res1, self._res_to_2darray(self._d_t, self.d_y,
-                                                         self.conf_int_mean(alpha=alpha)[1], decimals)))
+            res1 = np.hstack((res1, self._format_res(self.stderr_mean, decimals)))
+            res1 = np.hstack((res1, self._format_res(self.zstat(value=value), decimals)))
+            res1 = np.hstack((res1, self._format_res(self.pvalue(value=value), decimals)))
+            res1 = np.hstack((res1, self._format_res(self.conf_int_mean(alpha=alpha)[0], decimals)))
+            res1 = np.hstack((res1, self._format_res(self.conf_int_mean(alpha=alpha)[1], decimals)))
```

Comment:
I realize this was here already, but seems weird to continually restack instead of doing it all at once:

```suggestion
            res1 = np.hstack((res1, 
                              self._format_res(self.stderr_mean, decimals),
                              self._format_res(self.zstat(value=value), decimals),
                              self._format_res(self.pvalue(value=value), decimals),
                              self._format_res(self.conf_int_mean(alpha=alpha)[0], decimals),
                              self._format_res(self.conf_int_mean(alpha=alpha)[1], decimals)))
```